package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html

```diff
@@ -116,6 +130,17 @@ <h2><%:Overview%></h2>
 					<dt><%:Firmware%></dt><dd><%| nodeinfo.software.firmware.release %></dd>
 					<% if nodeinfo.software.fastd then -%>
 						<dt><%:Mesh VPN%></dt><dd><%| enabled(nodeinfo.software.fastd.enabled) %></dd>
+						<% if nodeinfo.network.mesh_vpn and nodeinfo.network.mesh_vpn.bandwidth_limit.enabled then -%>
+						<dt><%:Bandwidth limit%></dt>
+						<dd>
+							<% if nodeinfo.network.mesh_vpn.bandwidth_limit.egress then -%>
+							▲ <%| formatBits(nodeinfo.network.mesh_vpn.bandwidth_limit.egress, 1000) %>/s egress<br />
```

Comment:
Ingress/egress is incoherent with the config mode. There it is called Up/Downstream.

Reply:
Yep, both terms are used in gluon. E- and ingress are uci-terms.
You two outvote my opinion on this though; I'll implement it.

The line will then become `▼ 4 Downstream (Mbit/s)`.

> In general, I probably favor the terms used in config mode.

_Originally posted by @herbetom in https://github.com/freifunk-gluon/gluon/issues/2217#issuecomment-835493203_

Reply:
@AiyionPrime I don't understand... Why do you want to introduce a term which is potentially confusing for the users? Why do you want to break with consistency? Or better say why do we have this discussion if two people already said it should be consistent?

Reply:
0a64f04a28a04a356e9033e69bee007c24ef6004

Reply:
@CodeFetch You really need to start reading comments above.
> I'll implement it.

_Originally posted by @AiyionPrime in https://github.com/freifunk-gluon/gluon/pull/2217#discussion_r645821640_